core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/chain/UpgradeDatalakeFlowEventChainFactory.java

```diff
@@ -20,7 +20,7 @@
 @Component
 public class UpgradeDatalakeFlowEventChainFactory implements FlowEventChainFactory<ClusterUpgradeTriggerEvent> {
 
-    @Value("${cb.upgrade.validation.enabled:true}")
+    @Value("${cb.upgrade.validation.sdx.enabled:false}")
```

Comment:
Can you please remove the default value from here and move to app.yml? You might need to add it to multiple app.yml-s.